ckan/controllers/feed.py

```diff
@@ -145,7 +145,7 @@ def _create_atom_id(resource_path, authority_name=None, date_string=None):
         # This is best we can do, and if the site_url is not set, then
         # this still results in an invalid feed.
         site_url = config.get('ckan.site_url', '')
-        return '/'.join([site_url, resource_path])
+        return urlparse.urljoin(site_url, resource_path)
```

Comment:
So then we do not pass `resource_path` to this function anymore but pass some more data and create the actual path (or rather full url) in this function?